controllers/admin/AdminLegacyLayoutController.php

```diff
@@ -197,7 +197,7 @@ public function initContent()
             'hideLegacyStoreContextSelector' => $this->container->get('prestashop.adapter.multistore_feature')->isUsed(),
         ];
 
-        if ($this->helpLink === false || !empty($this->helpLink)) {
```

Comment:
Why did you remove this.
it can be false or not empty, meaning if you put false you reset the help link and if you put a string it's ok.
I'm afraid of side effect, what was the phpstan error for this one?

Reply:
Fixed. Sorry.